backend/test/deal-observer.test.js

```diff
@@ -107,6 +107,64 @@ describe('deal-observer-backend', () => {
     await storeBlockEvent(data)
     assert.strictEqual(await countStoredActiveDeals(pgPool), 2n)
   })
+
+  it('handles one client piece stored twice in the same sector and separate sql updates', async () => {
```

Comment:
To me this test title isn't clear, does it

"handle one client piece stored twice in the same sector, and also handle separate sql updates"

or does it

"handle one client piece stored twice in the same sector, and stored in separate sql updates"?

Reply:
I think you want to say that here we test processing two events for one piece, at once. Below, you want to test processing two events for one piece, in succession.

What about these test names instead?

```js
it('simultaneously processes claims for a piece stored twice in the same sector')
```

and

```js
it('serially processes claims for a piece stored twice in the same sector')
```